source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -3,13 +3,7 @@
 Use VirtualBox\*
 ################
 
-This section explains how to run |CLOSIA| inside a `VirtualBox environment`_.
-
-Please ensure you have enabled `Intel® Virtualization Technology
-<http://www.intel.com/content/www/us/en/virtualization/virtualization-technology/intel-virtualization-technology.html>`_
-(Intel® VT) and `Intel® Virtualization Technology for Directed I/O
-<https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
-(Intel® VT-d) in your BIOS/UEFI firmware configuration.
+This section explains how to run |CLOSIA| inside a `VirtualBox environment`_ on different host systems.
```

Comment:
I'd recommend changing "This section explains how to run |CLOSIA| inside a `VirtualBox environment`_ on different host systems." to "This section explains how to run |CLOSIA| as guest on VirtualBox environmen.
This section shows how to deploy a pre-configured Clear Linux image as a guest on the VirtualBox hypervisor."